TheBoxTests/TheBoxTests.csproj

```diff
@@ -18,6 +19,7 @@
     </ItemGroup>
 
     <ItemGroup>
+      <ProjectReference Include="..\TheBoxApp\TheBoxApp.csproj" />
```

Comment:
In the last test (line 810) I test extension method - Compress, and that method is defined in TheBoxApp/PudelkoExtensions. I guess it would make more sense to move it into Lib project? (and rename)

Reply:
Yep